lib/markdown_document.rb

```diff
@@ -0,0 +1,35 @@
+class MarkdownDocument
+  YAML_FRONT_MATTER_REGEXP = /\A(---\s*\n.*?\n?)^((---|\.\.\.)\s*$\n?)/m
```

Comment:
It makes me sad that Redcarpet doesn't handle Markdown frontmatter 😢

Reply:
Right?!